cognite/powerops/data_classes/shop_file_config.py

```diff
@@ -21,12 +22,17 @@ class ShopFileConfig(BaseModel):
         "extra_data",
         "water_value_cut_file_reservoir_mapping",
         "water_value_cut_file",
+        "modulserie",
```

Comment:
I agree with @ddonukis , I think this should be
```suggestion
        "modul_series",
```